charts/trino/templates/service.yaml

```diff
@@ -24,4 +27,4 @@ spec:
   selector:
     app: {{ template "trino.name" . }}
     release: {{ .Release.Name }}
-    component: coordinator
+    component: coordinator
```

Comment:
Revert this - configure your text editor not to remove end of line at the end of file.

Reply:
Done